core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -460,6 +477,20 @@ class WorldScreen(val viewingCiv:CivilizationInfo) : CameraStageBaseScreen() {
         else viewingCiv
     }
 
+    private fun createFogOfWarButton(): TextButton {
+        val fogOfWarButton = TextButton("Fog of War", skin)
```

Comment:
.toTextButton(), for the translation. Also, add "Fog of War" as a line in the translations template.properties

Reply:
ok, got it.